### ISSUE TITLE
5.0.0.pre.3 breaks with `have_current_path` Capybara matcher

### ISSUE THREAD

**Issue by geoffharcourt** [percy/percy-capybara#167]
Hello Percy team!

The 5.0.0.pre.3 release of the package causes `stack too deep` error in specs that use Capybara's `have_current_path` matcher.

Stacktrace looks like this (until it runs out of space):

```

Failure/Error: expect(page).to have_current_path(new_user_session_path)
--
&nbsp; | &nbsp;
&nbsp; | SystemStackError:
&nbsp; | stack level too deep
&nbsp; | &nbsp;
&nbsp; | &nbsp;
&nbsp; | &nbsp;
&nbsp; | [Screenshot Image]: /app/tmp/screenshots/failures_r_spec_example_groups_authentication_through_google_auth_when_not_using_interims_or_clever_o_auth_when_the_user_is_not_found_by_email_redirects_to_new_user_session_path;_displaying_email_in_error_mess_102.png
&nbsp; | &nbsp;
&nbsp; | &nbsp;
&nbsp; | &nbsp;
&nbsp; | &nbsp;
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara.rb:429:in `specified_session'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara.rb:315:in `current_session'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:46:in `page'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `assert_current_path'
&nbsp; | # /bundle/ruby/2.7.0/gems/capybara-3.35.3/lib/capybara/dsl.rb:53:in `call'
```

**Comment by Robdel12** [comment]
Hey @geoffharcourt! Thanks for the issue, I was able to ship a fix in `5.0.0.pre.4`. Curious to hear how it goes :)

**Comment by geoffharcourt** [comment]
Hi @Robdel12 this unblocked us (test suite now runs). We filed a new issue, #169 about something further downstream. Hoping we can get onto the new hotness. Thanks for your fix here.
